docs/features/alerting.md

```diff
@@ -0,0 +1,77 @@
+# Alerting
+
+
+Connaisseur can send notifications on admission decisions to basically every REST
+endpoint that accepts JSON payloads.
+
+## Supported Interfaces
+
+Slack, Opsgenie and Keybase have preconfigured payloads that are ready to use.
+However, you can use the existing payload templates as an example how to model your
+own custom one.
+It is also possible to configure multiple interfaces for receiving
+alerts at the same time.
+
+## Configuration options
+
+Currently, Connaisseur supports alerting on either admittance of images, denial of images or both. These event categories can be configured independently of each other under the relevant category (i.e. `admit_request` or `reject_request`):
+
+| key                                                |  accepted values                                      | default           | required           | description                                                                                        |
+| -------------------------------------------------- | ----------------------------------------------------  | ----------------- | ------------------ | -------------------------------------------------------------------------------------------------- |
+| `alerting.cluster_identifier`                      | string                                                | `"not specified"` |                    | cluster identifier used in alert payload to distinguish between alerts from different clusters     |
+| `alerting.<category>.template`                     | `opsgenie`, `slack`, `keybase` or custom<sup>*</sup>  | -                 | :heavy_check_mark: | specifies which file in `helm/alert_payload_templates/` to use as alert payload template           |
+| `alerting.<category>.receiver_url`                 | string                                                | -                 | :heavy_check_mark: | url of alert-receiving endpoint                                                                    |
+| `alerting.<category>.priority`                     | int                                                   | `3`               |                    | priority of alert (to enable fitting Connaisseur alerts into alerts from other sources)            |
+| `alerting.<category>.custom_headers`               | list[string]                                          | -                 |                    | additional headers required by alert-receiving endpoint                                            |
+| `alerting.<category>.payload_fields`               | subyaml                                               | -                 |                    | enables specifying additional (`yaml`) key-value pairs to be appended to alert payload (as `json`) |
+| `alerting.<category>.fail_if_alert_sending_fails`  | bool                                                  | `False`           |                    | `True` will make Connaisseur deny images if the corresponding alert cannot be successfully sent    |
```

Comment:
Other config bools are false/true instead of False/True. I guess this is due to how it is handled under the hood. Would that be something to change at some point? Out of scope for this PR, but if you think reasonable, maybe new issue

Reply:
feel free to create an issue